website/docs/cloud-docs/run/remote-operations.mdx

```diff
@@ -115,6 +115,8 @@ For full details about the stages of a run, see [Run States and Stages][].
 
 ## Import
 
-Terraform Cloud does not support remote execution for `terraform import`. For this command the workspace acts only as a remote backend for Terraform state, with all execution occurring on your own workstations or continuous integration workers.
+Terraform Cloud does not support remote execution for the `terraform import` command. For this command the workspace acts only as a remote backend for Terraform state, with all execution occurring on your own workstations or continuous integration workers.
 
 Since `terraform import` runs locally, environment variables defined in the workspace are not available. Any environment variables required by the provider you're importing from must be defined within your local execution scope.
+
+We recommend using [`import` blocks](/terraform/language/import), introduced in Terraform 1.5, to import resources in Terraform Cloud.
```

Comment:
Should we move this to the top of the import section since it's the recommended way? That way users won't read the first sentence and then give up.

Reply:
Great call!